python/chronos/docker/chronos_unitest_docker/README.md

```diff
@@ -0,0 +1,59 @@
+# Chronos nightly build docker image
+This dockerfile helps user to build a docker image where Chronos nightly build version is deploied.
+
+## Build an image
+First `cd` to this directory. Then build your docker image with Dockerfile:
+```bash
+sudo docker build -t chronos-nightly:b1 . # You may choose any NAME:TAG you want.
+```
+Or build with a proxy:
+```bash
+# typically, you need a proxy for building since there will be some downloading.
```

Comment:
typically, users don't need a proxy since their server/laptop/desktop is connected to Internet